README.md

```diff
@@ -13,3 +13,26 @@ This guide provides a uniform way to structure your RiotJS code. Making it
 * easier to cache and serve bundles of code separately.
 
 This guide is inspired by [AngularJS Style Guide](https://github.com/johnpapa/angular-styleguide) by John Papa.
+
+## Table of Contents
+
+1. [Component / Tag names](#component-tag-names)
+
+## Component / Tag names
```

Comment:
In response to 2#discussion_r52841439, since a Riot tag is a specific type of module, I'd prefer to stick with the term "module" when we are discussing something generic (1 module = 1 folder), use "tag" if talking about the actual custom element, and "tag module" when we mean the whole bundle the custom element is part of. So in your case, I would call the section "Tag module names". Overthinking it? :)